js/src/components/customer-effort-score-prompt/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import CustomerEffortScore from '@woocommerce/customer-effort-score';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { LOCAL_STORAGE_KEYS } from '.~/constants';
+import localStorage from '.~/utils/localStorage';
+
+/**
+ * CES prompt snackbar open
+ *
+ * @event gla_ces_snackbar_open
+ */
+/**
+ * CES prompt snackbar closed
+ *
+ * @event gla_ces_snackbar_closed
+ */
+/**
+ * CES modal open
+ *
+ * @event gla_ces_modal_open
+ */
+/**
+ * CES feedback recorded
+ *
+ * @event gla_ces_feedback
+ */
+
+/**
+ * A CustomerEffortScore wrapper that uses tracks to track the selected
+ * customer effort score.
+ *
+ * @fires gla_ces_snackbar_open whenever the CES snackbar (notice) is open
+ * @fires gla_ces_snackbar_closed whenever the CES snackbar (notice) is closed
+ * @fires gla_ces_modal_open whenever the CES modal is open
+ * @fires gla_ces_feedback whenever the CES feedback is recorded
+ *
+ * @param {Object} props React component props.
+ * @param {string} props.eventContext Context to be used in the CES wrapper events.
+ * @param {string} props.label Text to be displayed in the CES notice and modal.
+ * @return {JSX.Element} Rendered element.
+ */
+const CustomerEffortScorePrompt = ( { eventContext, label } ) => {
```

Comment:
💅  Non blocking

I guess we can add here

`if ( ! isWCTracksEnabled() ) return null;`  Since it is a common check for all the prompts, right?

And then remove them in `js/src/dashboard/campaign-creation-success-guide/index.js` and `js/src/product-feed/index.js`